standard.tex

```diff
@@ -98,54 +103,42 @@ \subsection{PCB Design}
 \end{figure}
 
 \item Don't print PCBs at the 3D Printing Center in E5. These are typically poor quality.
+\item Make sure that soldermask clearance is set such that there is soldermask between component pads (pay attention to fine pitched components) and that there is no mask on the pads themselves.
+\item Thoroughly check the Gerber files for the board before you send them out for fabrication.
 
 \item Points that only apply to designs intended to be handsoldered:
```

Comment:
"handsoldered" or "hand-soldered"? Nitpick, but we should be consistent.

Reply:
hand_soldered